src/oop_ext/conftest.py

```diff
@@ -0,0 +1,21 @@
+# mypy: disallow-untyped-defs
+
+from pathlib import Path
+from typing import Any, TYPE_CHECKING
+
+import pytest
+
+if TYPE_CHECKING:
+    from ._type_checker_fixture import TypeCheckerFixture
+
+
+@pytest.fixture
+def type_checker(request: Any, tmp_path: Path) -> "TypeCheckerFixture":
```

Comment:
The usage of `Any` on request is on purpose? 
Looks like pytest 6.2.0 exports pytest.FixtureRequest
```suggestion
def type_checker(request: pytest.FixtureRequest, tmp_path: Path) -> "TypeCheckerFixture":
```

Reply:
Indeed, thanks!